gapic/src/main/com/google/gapid/views/CommandTree.java

```diff
@@ -123,7 +122,7 @@ public CommandTree(Composite parent, Models models, Widgets widgets) {
       models.profile.removeListener(this);
     });
 
-    search.addListener(Events.Search, e -> search(e.text, (e.detail & Events.REGEX) != 0));
+    eventsFilter.addListener(Events.FilterEvents, e -> filterEvents((e.detail & Events.HIDE_HOST_COMMANDS) != 0, (e.detail & Events.HIDE_BEGIN_END) != 0, (e.detail & Events.HIDE_DEVICE_SYNC) != 0));
```

Comment:
Do we have java formatters that limit line length?

Reply:
This can be multiline.
```suggestion
    eventsFilter.addListener(Events.FilterEvents, 
       e -> filterEvents(
         (e.detail & Events.HIDE_HOST_COMMANDS) != 0, 
         (e.detail & Events.HIDE_BEGIN_END) != 0, 
         (e.detail & Events.HIDE_DEVICE_SYNC) != 0
      )
    );
```

Reply:
Done